chunk 1: Tweak section headers (local change).
chunk 2: Set the default operating system name (local change).

Index: tmac/an.tmac
--- tmac/an.tmac.orig
+++ tmac/an.tmac
@@ -195,7 +195,7 @@
 .  ds an*section1 General Commands Manual\"
 .  ds an*section2 System Calls Manual\"
 .  ds an*section3 Library Functions Manual\"
-.  ds an*section4 Kernel Interfaces Manual\"
+.  ds an*section4 Device Drivers Manual\"
 .  ds an*section5 File Formats Manual\"
 .  ds an*section6 Games Manual\"
 .  ds an*section7 Miscellaneous Information Manual\"
@@ -306,7 +306,7 @@
 .
 .  ds an-extra1 "\\$3\"
 .  ie (\\n[.$] > 3)    .ds an-extra2 "\\$4\"
-.  el                  .ds an-extra2 \" empty; but .AT/.UC can override
+.  el                  .ds an-extra2 "OpenBSD\"
 .
 .  if '\\*[an-extra1]'' \{\
 .    ds an-msg .\\$0 missing third argument; suggest document\"
